packages/contracts/src/ubiquistick/UbiquiStickSale.sol

```diff
@@ -101,18 +101,20 @@ contract UbiquiStickSale is Ownable, ReentrancyGuard {
 
         if (remainingTokenCount < count) {
             count = remainingTokenCount;
+            paid = remainingTokenCount;
         }
         if (msg.value < count * price) {
+            paid = (msg.value * price) / price;
             count = msg.value / price;
         }
         if (MAXIMUM_PER_TX < count) {
+            paid = MAXIMUM_PER_TX;
```

Comment:
`paid` should be equal to the amount of funds paid, not to the number of max allowed tokens to buy